docs/technical_reference/costing/ion_exchange.rst

```diff
@@ -19,41 +38,134 @@ The following variables are constructed on the unit block (e.g., m.fs.unit.costi
 .. csv-table::
    :header: "Description", "Symbol", "Variable Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "variable_name", "1", ":math:`\text{dimensionless}`"
+   "Density of regenerant solution", ":math:`\rho_{regen}`", "``regen_soln_dens``", "1000", ":math:`\text{kg}\text{m}^{3}`"
```

Comment:
In #1394, we are replacing the "Default Value" column with "Index", so you can probably just adjust this now rather than waiting for the conflict to emerge.

Reply:
That's a good idea, I thought providing the default value was odd for some of these.